var/spack/repos/builtin/packages/py-breathe/package.py

```diff
@@ -38,6 +39,7 @@ class PyBreathe(PythonPackage):
     depends_on("py-sphinx", type=("build", "run"))
```

Comment:
pygments is no longer required

Reply:
Maybe they just forgot to leave it in the setup.py file? The `breathe.filetypes` definitely has `import pygments`, and `Pygments>=1.6` is listed in the `requirements/production.txt` which is loaded by `pip` in the breathe CI scripts.

Reply:
Can you add a comment explaining that the dep is missing?